### DISCUSSION TITLE
why only web

### DISCUSSION THREAD

**Discussion by sk92129** [daohoangson/flutter_widget_from_html#1291] [General]
Why does this plugin on pub.dev only indicate that it supports web?
https://pub.dev/packages/flutter_widget_from_html

I got it working on iOS and Android (simple stuff like html link, embedded video, and some h2 mark up).
Does the support on iOS and Android limited to other features that are not obvious?

**Comment by daohoangson** [comment]
The pana tool will remove platform support if one of the dependencies doesn't support that platform. You can see the score details here https://pub.dev/packages/flutter_widget_from_html/score.
